www/src/utils/getParagonComponentsTypes.js

```diff
@@ -0,0 +1,52 @@
+const MANUALLY_INCLUDED_COMPONENTS = {
```

Comment:
[curious] Why were these specific ones necessary to manually handle?

Reply:
I have just realized that in the `Card` after overriding we lost `Img`, `Title` and `Subtitle`. I add to the Card and remove `MANUALLY_INCLUDED_COMPONENTS `.
```
Card.Img = BaseCard.Img;
Card.Title = BaseCard.Title;
Card.Subtitle = BaseCard.Subtitle;
```